README.md

```diff
@@ -1 +1,191 @@
-# rerender-vs-surgical-updates
+# Overview
+
+In this example we have implemented using React, Solidjs and Svelte a demo similar to the one they did in the 2021 React Conf presenting React Forget.
```

Comment:
Back in 2021 the React core team presented a TODO like sample app to show how _React Forget_ works, we have mimic this example and implemented it in _React / SolidJS and Svelte_